app/src/main/java/org/luxoft/sdl_core/MainActivity.java

```diff
@@ -63,13 +68,21 @@ public void onClick(View view) {
                     start_sdl_button.setEnabled(false);
                     stop_sdl_button.setEnabled(true);
                 }
+
+                if (isBleSupported() && isBluetoothPermissionGranted()) {
+                    startService(new Intent(MainActivity.this, org.luxoft.sdl_core.BleCentralService.class));
+                }
             }
         });
 
         stop_sdl_button.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
                 StopSDL();
+
+                if (isBleSupported()) {
```

Comment:
@ychernysheva I think there should be the same condition as for startService

Reply:
Done in e8ca88e58483ac32df3c23ffc3c31eadcd4cab87